lighthouse-core/config/default-config.js

```diff
@@ -419,6 +421,8 @@ const defaultConfig = {
         {id: 'performance-budget', weight: 0, group: 'budgets'},
         {id: 'resource-summary', weight: 0, group: 'diagnostics'},
         {id: 'third-party-summary', weight: 0, group: 'diagnostics'},
+        // Not enabled yet.
+        // {id: 'legacy-javascript', weight: 0, group: 'diagnostics'},
```

Comment:
not even in here but without a group?

Reply:
what do you mean? the audit isn't ready but when it is this line will be uncommented

Reply:
i'm wondering if #10311 really should have been a "next-config" or "experimental-config" ...